force-app/main/default/lwc/staffAvailability/staffAvailability.js

```diff
@@ -61,7 +61,6 @@ export default class StaffAvailability extends LightningElement {
 
 	renderedCallback() {
 		console.log('RECORD ID: ', this.recordId);
```

Comment:
remove console.logs

Reply:
actually, there are many console logs in these lwc components
I thought of not removing these as they might be left by developers intentionally.
I can do a commit of removing all console logs from all related lwc comps if you agree?